diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/chapter2/kdoc/KdocMethods.kt

```diff
@@ -251,12 +251,27 @@ class KdocMethods(configRules: List<RulesConfig>) : DiktatRule(
         }
     }
 
-    private fun ASTNode.isSingleLineGetterOrSetter() = isGetterOrSetter() && (expressionBodyTypes.any { hasChildOfType(it) } || getBodyLines().size == 1)
+    private fun ASTNode.isSingleLineGetterOrSetter(): Boolean {
+        val isDotQualifiedExp = this.findChildByType(DOT_QUALIFIED_EXPRESSION)?.psi
```

Comment:
You can cast it to a required type right away, e.g. using `?.let { it as KtDorQualifiedExpression }`

Reply:
Fixed